internal/entfilter/filter.go

```diff
@@ -184,9 +185,11 @@ func validPtrValue(ptr any) bool {
 	}
 
 	switch t := ptr.(type) {
-	case *enums.SIPStatus:
+	case *storage_enums.AIPStatus:
 		return t != nil && t.IsValid()
-	case *enums.PreprocessingTaskOutcome:
+	case *ingest_enums.SIPStatus:
+		return t != nil && t.IsValid()
+	case *ingest_enums.PreprocessingTaskOutcome:
```

Comment:
At some point we may want to remove this type of checks from the now general purpose `entfilter` package, I see it as a good candidate for our `go-tools` repository in the future.